frontend/src/components/LandingPage/Weather/Weather.js

```diff
@@ -0,0 +1,22 @@
+import React from 'react';
+import './Weather.css'
+
+const weatherData = {
+    city: "Allen" + " ",
```

Comment:
Not sure if there is a need for separating the two parts; ESLint is throwing warnings for this line.

Reply:
Nevermind, I think it's for connecting with API.